Sources/CloudWatchClient/AWSCloudWatchClient.swift

```diff
@@ -43,7 +43,7 @@ public enum CloudWatchClientError: Swift.Error {
 
     public func isRetriable() -> Bool? {
         switch self {
-        case .limitExceededException, .limitExceededFault:
+        case .limitExceededException, .limitExceededFault, .throttling:
```

Comment:
So we were not retrying on throttling errors before?

Reply:
Not for this client. We did have a level of retry on CW stream.